front/src/modules/pathfinding/components/Itinerary/ModalSuggestedVias.tsx

```diff
@@ -32,7 +35,10 @@ const ModalSuggestedVias = ({ suggestedVias }: ModalSuggestedViasProps) => {
     [destination]
   );
 
-  const removeViaFromPath = (op: SuggestedOP) => dispatch(removeVia(op));
+  const removeViaFromPath = (op: SuggestedOP) => {
+    const newPathSteps = pathSteps.filter((step) => !step || matchPathStepAndOp(step, op));
+    launchPathfinding(newPathSteps);
```

Comment:
Hum no, unfortunately, we need to relaunch the pathfinding every time we delete a via, because maybe it modifies the path found by the algorithm.

Here is an example of workflow which will be broken:
- I create a path between A and C
- I add a via B on the map which changes the pathfinding
- the suggested OPs contain now B'
- I add a via on B'
- I remove B, the pathfinding is launched again and is still the same thanks to B'
- I remove B', but since it was added from the suggested OPs, the pathfinding is not relaunched
- however, if I refresh the page, the path will be different (initial path between A and C)